packages/core/src/node/process/terminal-process.ts

```diff
@@ -38,6 +39,7 @@ export class TerminalReadableStream extends stream.Readable {
 @injectable()
 export class TerminalProcess extends Process {
 
+    id: number;
```

Comment:
should be enough to declare it in `Process`

Reply:
yes